src/pydicom/dataelem.py

```diff
@@ -551,6 +552,18 @@ def _convert(self, val: Any) -> Any:
         self.validate(val)
         return val
 
+    def __deepcopy__(self, memo: dict[int, Any] | None) -> "DataElement":
+        cls = self.__class__
+        copied = cls.__new__(cls)
+        if memo:
+            memo[id(self)] = copied
+
+        # Fix for #1816: don't deepcopy the parent!
+        for key in (k for k in self.__dict__ if k not in ("parent")):
+            copied.__dict__[key] = copy.deepcopy(self.__dict__[key], memo)
```

Comment:
`k not in ("parent")` should be `k != "parent"`.  As is, it's not even a tuple, just parentheses, so actually iterating the string?

Reply:
Ah, the (not) tuple was left over from triage. This still works, obviously, but you're right that it's not ideal.

Fixed